uniprot-config/src/main/java/org/uniprot/store/config/searchfield/common/AbstractSearchFieldConfig.java

```diff
@@ -159,4 +159,13 @@ private boolean isAnEvidenceType(SearchFieldItem fieldItem) {
         return EnumSet.of(SearchFieldType.EVIDENCE, SearchFieldType.EXPERIMENTAL_EVIDENCE)
                 .contains(fieldItem.getFieldType());
     }
+
+    private SearchFieldItem getSearchFieldItemByNameAndAlias(String fieldName) {
+        return this.findSearchFieldItemByName(fieldName)
```

Comment:
The name of the method should use "Or" since we are using either name or alias. And is telling that we are using both.

Reply:
Thank you. Updated